app/renderer/actions/Inspector.js

```diff
@@ -130,17 +130,22 @@ export function selectElement (path) {
     const [optimalStrategy, optimalSelector] = strategyMap.length > 0 ? strategyMap[strategyMap.length - 1] : ['xpath', selectedElementXPath];
 
     // Get the information about the element
-    const {elementId, variableName, variableType} = await callClientMethod({
+    let {elementId, variableName, variableType} = await callClientMethod({
       strategy: optimalStrategy,
       selector: optimalSelector,
     });
 
-    // Set the elementId, variableName and variableType for the selected element 
-    // (check first that the selectedElementPath didn't change, to avoid race conditions)
-    if (getState().inspector.selectedElementPath === path) {
-      dispatch({type: SET_SELECTED_ELEMENT_ID, elementId, variableName, variableType});
-    }
-    
+    if (!elementId) {
```

Comment:
IMHO positive conditions are more readable. Or we can do a fast return here, like

```
if (!element) {
   ....
   return;
}
```

Reply:
Agreed, yeah I changed it.